### DISCUSSION TITLE
Successfully finished Pytorch Fundamentals exercise

### DISCUSSION THREAD

**Discussion by sammigachuhi** [mrdbourke/pytorch-deep-learning#706] [Show and tell]
Alright, this may not move rocks but I was able to successfully finish the first exercise in the ztm pytorch-mastery.io course.

Here is the link to my colab: --- [pytorch fundamentals exercise](https://github.com/sammigachuhi/ml_projects/blob/main/pytorch_fundamentals_exercises.ipynb)

**Comment by shisirkha** [comment]
Wow that is amazing you completed Pytorch Fundamentals exercise.

This skill will help you to not get stuck.
